packages/rollup-plugin-import-meta-assets/test/snapshots/different-asset-levels-bundle.js

```diff
@@ -1,14 +1,14 @@
 const nameOne = 'one-name';
-const imageOne = new URL(new URL('assets/one-deep-Bkie7h0E.svg', import.meta.url).href, import.meta.url).href;
+const imageOne = new URL(new URL('assets/one-deep-Bkie7h0E.svg', import.meta.url).href).href;
```

Comment:
maybe I mislooked first time, but I was under impression that the whole outer URL wrapper will be gone with you change, not only the `import.meta.url`
do you think it's a more complex change or is the outer one still needed?